File: gcp-csharp/MyStack.cs
using Pulumi;
using Pulumi.Gcp.Storage;

class MyStack : Stack
{
    public MyStack()
    {
        // Create a GCP resource (Storage Bucket)
        var bucket = new Bucket("my-bucket");

        // Export the DNS name of the bucket
        this.BucketName = bucket.Url;
    }

    [Output]
    public Output<string> BucketName { get; set; }
}

File: gcp-visualbasic/MyStack.vb
Imports Pulumi
Imports Pulumi.Gcp.Storage

Class MyStack
    Inherits Stack

    Public Sub New()
        ' Create a GCP resource (Storage Bucket)
        Dim bucket = New Bucket("my-bucket")

        ' Export the DNS name of the bucket
        Me.BucketName = bucket.Url
    End Sub

    <Output>
    Public Property BucketName As Output(Of String)
End Class
